activerecord/lib/active_record/relation/query_methods.rb

```diff
@@ -1039,11 +1039,13 @@ def convert_join_strings_to_ast(table, joins)
       end
 
       def build_select(arel)
-        if select_values.any?
-          arel.project(*arel_columns(select_values.uniq))
-        else
-          arel.project(@klass.arel_table[Arel.star])
-        end
+        columns_to_select = if select_values.any?
+                              select_values.uniq
```

Comment:
Rubocop is asking this indentation. I believe because this rule is broken and needs to be fixed https://github.com/rails/rails/blob/master/.rubocop.yml#L116.